keywords/SyncGateway.py

```diff
@@ -683,18 +681,20 @@ def stop_sync_gateways(self, cluster_config, url=None):
         shut down the sync gateway at that url
         """
         ansible_runner = AnsibleRunner(cluster_config)
-
+        extra_vars = {}
```

Comment:
Is this being used? I see that the default argument for `ansible_runner.run_ansible_playbook()` is already an empty  dict, and it doesn't seem like this variable is ever changed?

Reply:
Thanks, fixed!